src/CodeExplorerBundle/EventListener/ControllerListener.php

```diff
@@ -12,22 +12,24 @@
 namespace CodeExplorerBundle\EventListener;
 
 use CodeExplorerBundle\Twig\SourceCodeExtension;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
 use Symfony\Component\HttpKernel\Event\FilterControllerEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
 
 /**
  * Defines the method that 'listens' to the 'kernel.controller' event, which is
  * triggered whenever a controller is executed in the application.
  *
  * See https://symfony.com/doc/current/book/internals.html#kernel-controller-event
  *
- * Tip: listeners are common in Symfony applications, but this particular listener
- * is too advanced and too specific for the demo application needs. For more common
- * examples see https://symfony.com/doc/current/cookbook/service_container/event_listener.html
+ * Tip: listeners and subscribers are common in Symfony applications, but this particular
+ * one is too advanced and too specific for the demo application needs.
+ * For more common examples see https://symfony.com/doc/current/event_dispatcher.html
```

Comment:
I would remove this extended doc. We already have it on the subscribers in the AppBundle. People digging into the internals of CodeExplorerBundle should already know what a subscriber is. And it will avoid having to maintain this message multiple time.